packages/liveblocks-react/src/liveblocks.tsx

```diff
@@ -299,10 +301,26 @@ function makeLiveblocksExtrasForClient(client: OpaqueClient) {
     { maxStaleTimeMs: config.USER_THREADS_MAX_STALE_TIME }
   );
 
+  const channelNotificationSettingsPoller = makePoller(
+    async (signal) => {
+      try {
+        return await store.refreshChannelNotificationSettings(signal);
+      } catch (err) {
+        console.warn(
+          `Polling new channel notification settings failed: ${String(err)}`
+        );
+        throw err;
+      }
+    },
+    config.CHANNEL_NOTIFICATION_SETTINGS_INTERVAL,
+    { maxStaleTimeMs: config.CHANNEL_NOTIFICATION_SETTINGS_MAX_TIME }
+  );
+
```

Comment:
I don't think this poller is really needed for your use case? We could remove it if you feel like it's relatively a lot of added complexity for little value, because I don't think notification settings are that likely to change (unlike threads and notifications, etc). When they do, is it really important that connected clients see it reflect immediately?

If you think the polling is nice to have, feel free to keep it in.
But if you feel it adds too much complexity for this use case, feel free to remove it.
At the very least, maybe we could lower the poll interval a bit for this resource, as I don't think it will be necessary to poll every minute. (The way the poller works anyway is that any time the window gets re-focused, it will trigger a poll anyway if the data is stale (as defined by the `MAX_STALE_TIME` value.)

Reply:
Yes and no 🙂 
I'm glad you're pointing this. To me we have to have it because we will offer methods in the `@liveblocks/node` client to update the channel notifications. 

So I think at some use cases where you could update the channel notification settings from the node client such as for example support request or when a user in an app enables an integration like Slack and the app needs to update the channel notification settings by itself and do not expect the user to refresh the app to see the new channel notification settings.

So I'd keep the poller for now because I think it's important to have but based on your suggestion below I will lower the interval value. 
However 10 mins since a bit too much from my perspective. I will tend to see something more in between like 6 mins instead. 

Thanks 🙏🏻